core/docker/bin/health-check

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function get_property() {
+    grep "^$1=" "$2" | cut -d'=' -f2
+}
+
+scheme=http
+port=8080
+
+config=/etc/trino/config.properties
+if [ "$(get_property 'http-server.https.enabled' "$config")" == true ]; then
+    scheme=https
+    port=$(get_property http-server.https.port "$config")
+else
+    port=$(get_property http-server.http.port "$config")
+fi
+
+endpoint="$scheme://localhost:$port/v1/info"
+
+# add --insecure to disable certificate verification in curl, in case a self-signed certificate is being used
+if ! info=$(curl --fail --silent --show-error --insecure "$endpoint"); then
```

Comment:
it's recommendable to `{}`-brace vars

Reply:
Is there a convention for shell scripts in Trino or is that just personal preference? I disagree in this case since it makes it less readable and there are no other usages of this syntax in this file.

Reply:
i follow google bash code style with this

whatever

Reply:
For reference: https://google.github.io/styleguide/shellguide.html

Too bad there's no formatter tool that'd apply this. I use `shfmt`.